2021q3/doceng.adoc

```diff
@@ -0,0 +1,28 @@
+////
+Quarter:	3rd quarter of 2021
+Prepared by:	dbaio
+Reviewed by:	gjb, carlavilla
+Last edit:	$Date: 2021-10-03 05:34:14 -0300 (Sun, 03 Oct 2021) $
+Version:	$Id: doceng-2021-3rd-quarter-status-report.adoc 185 2021-10-03 08:34:14Z carlavilla $
+////
+
```

Comment:
Assuming the block above is part of doceng workflow (and therefore way above my pay grade) and has a good reason to be there.

Reply:
This is just a comment.
if necessary, you can remove it.

Reply:
I'll let @debdrup decide.

Reply:
The metadata wasn't present in the doceng entry in the last status report, and nor do any of the other entries include similar metadata - so I don't see a reason to include it.